lib/echo/base.rb

```diff
@@ -54,5 +58,18 @@ def wrap_with_envelope(content)
         end
       end
     end
+
+    # sets the timeout used for faraday connections
+    def timeout=(value)
```

Comment:
We discussed and you commented about trying to test these timeout functions for the echo_client

Reply:
I wrote a small test to ensure the timeouts are all set in each delegate.   We should discuss more tomorrow if we need any other tests.   I sorted of hacked it a bit though, as @services wasn't exposed to the public as had to add a attr_reader to expose it to the publicto ensure all the timeouts where set.  Not sure I'm crazy about doing that though.